src/tailwind.js

```diff
@@ -86,7 +94,7 @@ module.exports = {
     },
   },
   variants: {
-    backgroundColor: ["responsive", "hover", "focus", "active"],
+    backgroundColor: ["responsive", "hover", "focus", "active", "even", "odd"],
     textColor: ["responsive", "hover", "focus", "active"],
     borderRadius: ["responsive", "hover", "focus", "active"],
   },
```

Comment:
so i realise this was accidentally overwriting the defaults, use extend something like this:

```
variants: {
    extend: {
      backgroundColor: ["active", "even", "odd"],
      textColor: ["active"],
      borderRadius: ["active"],
    },
  },
  ```
  
  checklist of what's in default -> https://tailwindcss.com/docs/hover-focus-and-other-states#default-variants-reference

Reply:
done.